changelogs/unreleased/5047-fix-contradicting-log-message.yml

```diff
@@ -0,0 +1,4 @@
+description: fix a contradicting message in the logs of the compiler service
+change-type: patch
+destination-branches: [iso5]
```

Comment:
Don't we have to merge this one on iso4 as well?

Reply:
indeed, stupid me wanted to create 2 separated PRs :p